Essentials/src/main/java/com/earth2me/essentials/UserMap.java

```diff
@@ -170,6 +170,30 @@ public User load(final String stringUUID) throws Exception {
         throw new Exception("User not found!");
     }
 
+    public User load(final org.bukkit.OfflinePlayer player) throws Exception {
+        if (player == null) {
+            throw new IllegalArgumentException();
```

Comment:
Might be good to add a message in this exception in case someone runs into it using this method